migrations/20200607190058-guild_table.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, DataTypes) => {
+    return queryInterface.createTable('Guild', {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+      },
+      region: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      realm: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      nameSlug: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: true,
```

Comment:
These fields no longer need to be nullable, since we won't have to wipe the data